lib/active_record_extended/query_methods/unionize.rb

```diff
@@ -86,15 +87,17 @@ def unionize_storage!
       end
 
       {
-        union_values:          Array,
-        union_operations:      Array,
-        union_ordering_values: Array,
-        unionized_name:        lambda { |klass| klass.arel_table.name }
```

Comment:
Thanks for fixing this! this is an issue on rails 7.1 and 7.2 as well.

There is still a reference to `@klass.arel_table` on line 147 : https://github.com/GeorgeKaraszi/ActiveRecordExtended/blob/84536e187642da6061b2494617719396218c12dd/lib/active_record_extended/query_methods/unionize.rb#L147